apps/mobile/.eslintrc.json

```diff
@@ -24,6 +29,13 @@
         // Okay to use require in these files
         "@typescript-eslint/no-var-requires": "off"
       }
+    },
+    {
+      "files": ["src/components/GalleryTouchableOpacity.tsx"],
+      "rules": {
+        // Only allow TouchableOpacity import in this file
+        "no-restricted-imports": "off"
+      }
```

Comment:
I would just inline ignore this in the file. This way we're not ignoring other restricted imports.